jest.config.js

```diff
@@ -1,41 +1,15 @@
-// For a detailed explanation regarding each configuration property, visit:
-// https://jestjs.io/docs/en/configuration.html
+const jestConfig = require('./packages/jest-config-terra');
 
 module.exports = {
-  clearMocks: true,
-  collectCoverage: true,
-  collectCoverageFrom: [
-    'packages/**/src/**/*.js',
-    '!packages/terra-cli/tests/**/*.js',
-    '!packages/terra-functional-testing/src/services/wdio-visual-regression-service/commands/*.js',
-    '!packages/terra-functional-testing/src/services/wdio-visual-regression-service/methods/(S)*.js',
-    '!packages/terra-functional-testing/src/services/wdio-visual-regression-service/modules/(a|b|m)*.js',
-    '!packages/terra-functional-testing/src/services/wdio-visual-regression-service/scripts/*.js',
-    '!packages/terra-functional-testing/src/services/wdio-visual-regression-service/utils/**/*.js',
-  ],
+  ...jestConfig,
+  testEnvironment: './packages/jest-config-terra/lib/JestEnvironmentJsdomTerra.js',
```

Comment:
in the config in terra-toolkit we have to override more because we normally resolve to the node modules directory instead of the package directory